eagle-core/eagle-alert-parent/eagle-alert/pom.xml

```diff
@@ -291,6 +295,32 @@
                 <artifactId>commons-cli</artifactId>
                 <version>${common.cli.version}</version>
             </dependency>
+            <dependency>
+            	<groupId>org.mongodb</groupId>
+	            <artifactId>mongo-java-driver</artifactId>
+	            <version>${mongodb.version}</version>
+	        </dependency>
+	        <dependency>
+				<groupId>org.codehaus.jackson</groupId>
+				<artifactId>jackson-jaxrs</artifactId>
+				<version>${codehaus-jackson.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.codehaus.jackson</groupId>
+				<artifactId>jackson-xc</artifactId>
+				<version>${codehaus-jackson.version}</version>
+			</dependency>
+	        <dependency>
+				<groupId>org.codehaus.jackson</groupId>
+				<artifactId>jackson-mapper-asl</artifactId>
+				<version>${codehaus-jackson.version}</version>
```

Comment:
Please try to use `fasterxml-jackson` instead of `codehaus-jackson` anytime if possible.